src/content/docs/tools/cli.mdx

```diff
@@ -5,6 +5,11 @@ sidebar:
   label: Daytona CLI Reference
 ---
 import Aside from "@components/Aside.astro";
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Cloud**, **Self-Managed**
```

Comment:
This is a bit problematic since the CLI is much different in Open Source and EE.